File: BEmuJava/bemu/src/main/java/com/bemu/BEmu/SessionOptions.java
//------------------------------------------------------------------------------
// <copyright project="BEmuJava" file="BEmu.SessionOptions.java" company="Jordan Robinson">
//     Copyright (c) 2013 Jordan Robinson. All rights reserved.
//
//     The use of this software is governed by the Microsoft Public License
//     which is included with this distribution.
// </copyright>
//------------------------------------------------------------------------------

package com.bemu.BEmu;

public class SessionOptions {
	@SuppressWarnings("unused")
	private String _serverhost;
	public void setServerHost(String host)
	{
		this._serverhost = host;
	}
	
	@SuppressWarnings("unused")
	private int _serverPort;
	public void setServerPort(int port)
	{
		this._serverPort = port;
	}
}

File: BEmu_cpp/src/BloombergTypes/SessionOptions.cpp
//------------------------------------------------------------------------------------------------
// <copyright project="BEmu_cpp" file="src/BloombergTypes/SessionOptions.cpp" company="Jordan Robinson">
//     Copyright (c) 2013 Jordan Robinson. All rights reserved.
//
//     The use of this software is governed by the Microsoft Public License
//     which is included with this distribution.
// </copyright>
//------------------------------------------------------------------------------------------------

#include "BloombergTypes/SessionOptions.h"

namespace BEmu
{
	void SessionOptions::setServerPort(unsigned short port)
	{
		this->_port = port;
	}

	void SessionOptions::setServerHost(const char * host)
	{
		this->_host = host;
	}

	const char* SessionOptions::serverHost() const
	{
		return this->_host;
	}

	unsigned short SessionOptions::serverPort() const
	{
		return this->_port;
	}

	SessionOptions::SessionOptions()
	{
	}
}